hieradata/clients/aws.ci.jenkins.io.yaml

```diff
@@ -60,13 +60,13 @@ profile::jenkinscontroller::jcasc:
         installers:
           linux-arm64:
             type: "zip"
-            label: "linux-arm64 && arm64"
+            label: "(linux-arm64 && arm64) || (linux && arm64)"
```

Comment:
the right hand-side makes the most sense from a label selector point of view ( `linux && arm64`)

`linux-arm64 && arm64` seems weird, linux-arm64 already requires arm64, why is arm64 required as well.

Reply:
Good point. I chose to be conservative and only focused on the things added, but what you underline totally make sense. Let me update the PR

Reply:
https://github.com/jenkins-infra/jenkins-infra/pull/4323/commits/8e335eab43627d1799ad011d3b962599526aad8c